### ISSUE TITLE
A11Y, NumberInput: Form controls must have associated labels

### ISSUE THREAD

**Issue by emmafull** [IBM/carbon-components-react#1490]
## Detailed description

> Describe in detail the issue you're having.

The up and down arrow buttons in NumberInput need to have associated labels for accessibility. 

===== ENVIRONMENT =====
IBM CI 162 Checkpoint Violation: IBM 1.3.1 Info and Relationships
IBM Guidance: https://www.ibm.com/able/guidelines/ci162/info_and_relationships.html
Test Tool Type: DAP

===== DEFECT DESCRIPTION =====
The rule detected an unlabeled interactive component.

Each Form control must have an associated label. The label requirement applies to standard form controls, interactive elements and custom components.

===== EXPECTED RESULT =====
https://aat.w3ibm.mybluemix.net/token/334ed6c7-0a0d-480b-a007-47d1242d0ef8/1c0df9a2-5788-4caa-8052-67d0bcadfdac/archives/2018AugustDeploy//doc/w3/help/en-US/idhi_accessibility_check_g41.html

> Is this issue related to a specific component?

NumberInput

> What did you expect to happen? What happened instead? What would you like to see changed?

Expected the up and down arrow icons to have an associated label for accessibility. No labels are present. It would be great if `aria-label`s are added to the up and down buttons.

> What browser are you working in?

Chrome v69

> What version of the Carbon Design System are you using?

"carbon-components-react": "5.52.4"

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IBM Cloud Internet Services

## Additional information

<img width="876" alt="arrow-up" src="https://user-images.githubusercontent.com/11358963/47593634-9ae91880-d93d-11e8-84c6-ca2dc6db6c14.png">
<img width="821" alt="arrow-down" src="https://user-images.githubusercontent.com/11358963/47593642-a1779000-d93d-11e8-9ed6-8b7472ff8a1a.png">

**Comment by joshblack** [comment]
@emmafull I believe this is a false positive coming from DAP. The `button` itself for the icon isn't a `form` control (has `type="button"`), and it should announce the "contents" of the button as the label, see:

![screen shot 2018-10-26 at 5 05 26 pm](https://user-images.githubusercontent.com/3901764/47594529-5b243000-d941-11e8-8943-07b0127fddc7.png)

**Comment by emmafull** [comment]
@joshblack Thank you very much for your explanation. This is accepted by the Accessibility Verification Team and the issue can be closed.
